base/Dockerfile.ubi8

```diff
@@ -43,19 +43,20 @@ ENV LANG="C.UTF-8"
 ENV CUB_CLASSPATH='"/usr/share/java/cp-base-new/*"'
 
 # Zulu openJDK
-ENV ZULU_OPENJDK="zulu-11-11.35+15-1"
+ENV ZULU_OPENJDK="zulu-11-11.99+0-1"
 
 ENV PYTHON_VERSION="36-3.6.8"
 
 COPY requirements.txt .
 
 RUN microdnf install yum \
-    && yum update -q -y \
+    && yum update -y \
     && yum install -y git wget nc python${PYTHON_VERSION} tar procps krb5-workstation iputils hostname \
     && alternatives --set python /usr/bin/python3 \
     && pip3 install --only-binary --install-option="--prefix=/usr/local" --upgrade -rrequirements.txt \
-    && rpm --import http://repos.azulsystems.com/RPM-GPG-KEY-azulsystems \
-    && curl -o /etc/yum.repos.d/zulu.repo http://repos.azulsystems.com/rhel/zulu.repo \
+    # https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/PrepareZuluPlatform/AttachYumRepositoryRHEL-SLES-OracleLinuxSys.htm
+    && rpm --import https://www.azul.com/files/0xB1998361219BD9C9.txt \
+    && yum -y install https://cdn.azul.com/zulu/bin/zulu-repo-1.0.0-1.noarch.rpm \
```

Comment:
Do we know what is in this `zulu-repo...rpm`? 
I am not too familiar with poking inside rpm files, but my guess is that this is just a meta package that sets up the right `yum` repositories for zulu (i.e. what `cp-server-connect-base` is doing).

Reply:
I'm not sure why they chose to implement it as a post-script, but this is the "contents":

```
aegelhofer@Andrew-Egelhofer's- tst % rpm -q --scripts -p zulu-repo-1.0.0-1.noarch.rpm
warning: zulu-repo-1.0.0-1.noarch.rpm: Header V4 RSA/SHA256 Signature, key ID 219bd9c9: NOKEY
postinstall scriptlet (using /bin/sh):
if [ -d /etc/yum.repos.d ]; then
    reposdir=/etc/yum.repos.d
elif [ -d /etc/zypp/repos.d ]; then
    reposdir=/etc/zypp/repos.d
else
    exit 0
fi
cat > ${reposdir}/zulu-openjdk.repo <<EOF
[zulu-openjdk]
name=zulu-openjdk - Azul Systems Inc., Zulu packages
baseurl=https://repos.azul.com/zulu/rpm
gpgkey=https://assets.azul.com/files/0xB1998361219BD9C9.txt
enabled=1
gpgcheck=1
protect=1
EOF
exit 0
```

Reply:
Ah yes.  Exactly what I expected. 
Probably because they need to `compute` the correct path, for which some executable needs to run somewhere.